ui/src/components/VhostsListView.vue

```diff
@@ -95,6 +95,9 @@
             <div class="list-group-item-text">
               <div :class="[item.status === 'disabled' ? 'gray':'']">{{ item.Description }}</div>
             </div>
+            <div class="list-group-item-text">
+              <div :class="[item.status === 'disabled' ? 'gray':'']">{{(item.PhpRhVersion ==='default')? 'PHP54' : item.PhpRhVersion.toUpperCase()}}</div>
```

Comment:
Can we use a more "traditional" string, with spaces and periods? The `$t()` translation function might help...

```suggestion
              <div :class="[item.status === 'disabled' ? 'gray':'']">{{ $t('virtualhost.php_version_' + item.PhpRhVersion) }}</div>
```

Reply:
done